src/easydmp/dmpt/admin.py

```diff
@@ -120,29 +120,29 @@ def is_published(self, obj):
         if obj.published:
             return True
         return False
-    is_published.short_description = 'Is published'
-    is_published.boolean = True
+    is_published.short_description = 'Is published'  # type: ignore
+    is_published.boolean = True  # type: ignore
```

Comment:
@hmpf What is the implication of not having all these? I've seen many codebases with mypy, but I don't recall seeing these used everywhere.

Reply:
Django obviously uses these magical attributes all over the place, as do I in for instance the `LoginRequiredMiddleware`, checking if `login_required` has been set, and it's value. Functions (and methods) are objects and you can set arbitrary attributes on an object as long as it haven't set `__slots__` IIRC. So handy, very pythonic.

AFAIK there's no way to annotate that these extra attributes are allowed or I am sure Django would have done so already. Therefore, since mypy do not understand these, I turn off the errors. (I was tempted to turn off type checking for all admin-files because of this, it's so common in admin.)

It would be nice if there *was* a way to add annotations for them, they are so easy to mistype. I don't want to get involved with core python dev but maybe if there was a magical dunder-attribute with a list of extra attributes and mypy could check that? `__optional_attributes__`, say.